src/components/segmentedControl/segmentedControl.svelte

```diff
@@ -169,35 +169,35 @@
         left 0.4s cubic-bezier(0.22, 1, 0.36, 1);
     }
 
-    :where(&) > :global .leo-control-item,
-    :where(&) > :global ::slotted(leo-controlitem) {
+    :where(&) > :global(.leo-control-item),
```

Comment:
Does this do anything when we're in a WebComponent?

Reply:
You mean does it break anything?

Reply:
I mean, does this component work as a webcomponent properly - I don't think these selectors will work

Reply:
The selector is working in the `plain-html` example (the greyed out variable names seems to be a dev tools quirk with the shadow dom).

![SCR-20250310-kvxm-2](https://github.com/user-attachments/assets/357fc874-025e-4f93-aac3-64de69d9854e)

Reply:
Sweet!